src/shortcuts/__tests__/redo.ts

```diff
@@ -49,20 +49,23 @@ it('group contiguous navigation actions preceding a thought change on redo', ()
   const store = createTestStore()
 
   store.dispatch([importText(RANKED_ROOT, `
-      - a
-      - b`
+    - a
+    - b`
   ),
   { type: 'cursorDown' },
+  { type: 'setCursor', thoughtsRanked: [{ value: 'b', rank: 1 }] },
+  { type: 'indent' },
   {
     type: 'existingThoughtChange',
     newValue: 'ar',
     oldValue: 'a',
     context: [ROOT_TOKEN],
     thoughtsRanked: [{ value: 'a', rank: 0 }]
   },
-  { type: 'cursorUp' },
-  { type: 'setCursor', thoughtsRanked: [{ value: 'ar', rank: 0 }] },
   { type: 'cursorBack' },
+  { type: 'cursorUp' },
+  { type: 'setCursor', thoughtsRanked: [{ value: 'b', rank: 0 }] },
```

Comment:
`[{ value: 'b', rank: 0 }]` is not a valid `Path` since `b` was indented to become `[{ value: 'a', rank: 0 }, { value: 'b', rank: 0 }]`.